e2e/playwright/info-drawer/src/tests/file-folder-properties.e2e.ts

```diff
@@ -132,18 +132,6 @@ test.describe('Info Drawer - File Folder Properties', () => {
     await personalFiles.acaHeader.viewDetails.click();
   }
 
-  async function waitForTagToBeAdded(folderId: string, personalFiles: PersonalFilesPage, maxRetries: number) {
-    let retries = 0;
-
-    while ((await tagsApi.listTagsForNode(folderId)).list.entries.length === 0) {
```

Comment:
listTagsForNode is unused function now so can be removed

Reply:
I would leave it because it's in the tagsApi, so it can be used in the future. Its not a custom method, its just an api call from swagger.